selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java

```diff
@@ -241,17 +245,15 @@ public void waitProjectExplorer() {
   }
 
   /**
-   * wait appearance of the IDE Project Explorer
+   * Waits appearance of the IDE Project Explorer
    *
-   * @param timeout user timeout
+   * @param timeout waiting timeout in seconds
    */
   public void waitProjectExplorer(int timeout) {
```

Comment:
Lets name it just `wait`, because `waitProjectExplorer` duplicates class name **ProjectExplorer**.

Reply:
Sorry, `wait()` can't be used indeed. `waitAppearance()` is possible variant instead, IMHO